doc/libpmemobj.3.md

```diff
@@ -3,7 +3,7 @@ layout: manual
 Content-Style: 'text/css'
 title: libpmemobj(3)
 header: NVM Library
-date: pmemobj API version 2.0.0
+date: pmemobj API version 2.0.1
```

Comment:
Right, it's not a bug fix, but it is backward-compatible. I'll change to 2.1.0. (same for API version).